spec/apps/dummy/app/controllers/mock_groups_controller.rb

```diff
@@ -1,4 +1,4 @@
-class MockUsersController < Scimitar::ActiveRecordBackedResourcesController
+class MockGroupsController < Scimitar::ActiveRecordBackedResourcesController
```

Comment:
Controller was never previously used by tests, so this error had gone unnoticed. Now that both User and Group listing and showing is used for testing a standard name, integer primary key vs a renamed, UUID primary key, the error came to light.